app/views/registered_users/boards/show.html.erb

```diff
@@ -1,14 +1,15 @@
 <section class="board">
-<% if current_user == @board_presenter.user %>
-<%= link_to 'Edit', registered_users_edit_board_path(@board_presenter.user.username, @board_presenter.board.name)%>
-<% end %>
-<% if current_admin %>
-<%= link_to "Delete", admin_board_path(@board_presenter.board), method: :delete %>
-<% end %>
-<%= @board_presenter.board.name %>
-<%= @board_presenter.board.description %>
-<%= @board_presenter.board.privacy %>
-<%= @board_presenter.user.name %>
+  <% if current_user == @board_presenter.user %>
```

Comment:
Maybe we can get together to brainstorm how to remove this authorization and my authorization out of the views. Looks fine for now, though!